test/unit/logic/round.js

```diff
@@ -17,153 +17,129 @@
+		scope = _.cloneDeep(validScope);
+		round = new Round(_.cloneDeep(scope), dbStub);
+	});
+
+	afterEach(async () => {
+		dbStub.rounds.updateVotes.reset();
+		dbStub.rounds.updateDelegatesRanks.reset();
+		dbStub.rounds.restoreRoundSnapshot.reset();
+		dbStub.rounds.restoreVotesSnapshot.reset();
+		dbStub.rounds.checkSnapshotAvailability.reset();
+		dbStub.rounds.countRoundSnapshot.reset();
+		dbStub.rounds.deleteRoundRewards.reset();
+		dbStub.rounds.insertRoundRewards.reset();
+		dbStub.rounds.updateMissedBlocks.reset();
+
+		storageStub.entities.Round.delete.reset();
+		storageStub.entities.Round.getTotalVotedAmount.reset();
+		scope.modules.accounts.mergeAccountAndGet.reset();
+		sinonSandbox.restore();
 	});
 
 	function isPromise(obj) {
 		return typeof obj.then === 'function';
 	}
 
 	describe('constructor', () => {
-		let scope;
-
 		describe('when calling with required properties', () => {
-			before(done => {
-				scope = _.cloneDeep(validScope);
-				round = new Round(_.cloneDeep(scope), db);
-				done();
-			});
-
 			it('should return Round instance', () => {
 				return expect(round).to.be.instanceof(Round);
 			});
 
 			it('should set scope', () => {
-				return expect(round.scope).to.deep.equal(scope);
+				return expect(round.scope).to.be.eql(validScope);
 			});
 
 			it('should set t', () => {
-				return expect(round.t).to.deep.equal(db);
+				return expect(round.t).to.be.eql(dbStub);
 			});
 		});
 
 		describe('when calling with missing properties', () => {
-			beforeEach(done => {
-				scope = _.cloneDeep(validScope);
-				done();
-			});
-
-			/* eslint-disable mocha/no-skipped-tests */
-			describe.skip('library', () => {
-				it('should throw', done => {
-					const property = 'library';
-					delete scope[property];
-					try {
-						round = new Round(_.cloneDeep(scope), db);
-					} catch (err) {
-						expect(err).to.equal(
-							`Missing required scope property: ${property}`
-						);
-					}
-					done();
-				});
-			});
-
-			describe.skip('modules', () => {
-				it('should throw', done => {
-					const property = 'modules';
-					delete scope[property];
-					try {
-						round = new Round(_.cloneDeep(scope), db);
-					} catch (err) {
-						expect(err).to.equal(
-							`Missing required scope property: ${property}`
-						);
-					}
-					done();
-				});
-			});
-
-			describe.skip('block', () => {
-				it('should throw', done => {
-					const property = 'block';
-					delete scope[property];
-					try {
-						round = new Round(_.cloneDeep(scope), db);
-					} catch (err) {
-						expect(err).to.equal(
-							`Missing required scope property: ${property}`
-						);
-					}
-					done();
-				});
-			});
-			/* eslint-enable mocha/no-skipped-tests */
-
 			describe('round', () => {
 				it('should throw', done => {
 					const property = 'round';
 					delete scope[property];
 					try {
-						round = new Round(_.cloneDeep(scope), db);
+						round = new Round(_.cloneDeep(scope), dbStub);
```

Comment:
Same here, as `scope` is already a deep clone of `validScope` (in `beforeEach`) there is no need to `cloneDeep` it again.